lms/templates/logout.html

```diff
@@ -5,14 +5,26 @@
 {% block title %}{% trans "Signed Out" as tmsg %}{{ tmsg | force_escape }} | {{ block.super }}{% endblock %}
 
 {% block body %}
-    <h1>{% trans "You have signed out." as tmsg %}{{ tmsg | force_escape }}</h1>
+    {% if enterprise_target %}
+        <h1>{% trans "We are signing you in." as tmsg %}{{ tmsg | force_escape }}</h1>
```

Comment:
@ziafazal Thanks for adding a user message here.  Can you also add a developer comment in this `if` clause.  Otherwise, by itself, it can be confusing why we are saying "We are signing you in" in a logout flow in `logout.html`.